### PR TITLE
Update installation guide

### PR DESCRIPTION
**Pull request by mogita**
Hi, just updated a production installation guide. Also updated some of the `.env.sample` fields to be more descriptive.

Please leave comments, thanks!

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -4,11 +4,11 @@
 #
 # Please use `openssl rand -hex 32` to create SECRET_KEY
 
-DATABASE_URL=postgres://user:pass@postgres:5432/outline
-DATABASE_URL_TEST=postgres://user:pass@postgres:5432/outline-test
-SECRET_KEY=F0E5AD933D7F6FD8F4DBB3E038C501C052DC0593C686D21ACB30AE205D2F634B
+DATABASE_URL=postgres://user:pass@localhost/outline
```
Comment (tommoor) [inline .env.sample]: We should leave the port in the example here

File: README.md [hunk 2]
```diff
@@ -33,6 +35,28 @@ In development you can quickly get an environment running using Docker by follow
    else should work well for development.
 1. Run `make up`. This will download dependencies, build and launch a development version of Outline.
 
+For production deployment by building from scratch, you can follow these steps:
+
+1. Clone this repo and install dependencies with `npm install` or `yarn`
```
Comment (tommoor) [inline README.md]: I think the installation guide should prefer `yarn`, as we use yarn for development and a yarn lockfile is present there's a small possibility that installing with NPM will result in a different set of dependencies being installed

File: .env.sample (hunk 1 above)
Comment (mogita) [reply .env.sample]: Fixed

File: README.md (hunk 2 above)
Comment (mogita) [reply README.md]: Fixed

**Comment by mogita** [comment]
Looks like a type check has failed. What's your idea @tommoor ? Do you need me to fix it for you?

**Comment by tommoor** [comment]
Master was failing for a couple of commits there, sorry about that – I'll fix that up and get this merged, thanks!

**Pull request merged**